src/field_theory/adjoin.lean

```diff
@@ -171,11 +178,25 @@ end
 lemma algebra_adjoin_le_adjoin : algebra.adjoin F S ≤ (adjoin F S).to_subalgebra :=
 algebra.adjoin_le (subset_adjoin _ _)
 
-lemma adjoin_le_algebra_adjoin (inv_mem : ∀ x ∈ algebra.adjoin F S, x⁻¹ ∈ algebra.adjoin F S) :
-  (adjoin F S).to_subalgebra ≤ algebra.adjoin F S :=
-show adjoin F S ≤
-  { neg_mem' := λ x, (algebra.adjoin F S).neg_mem, inv_mem' := inv_mem, .. algebra.adjoin F S},
-from adjoin_le_iff.mpr (algebra.subset_adjoin)
+lemma adjoin_eq_algebra_adjoin (inv_mem : ∀ x ∈ algebra.adjoin F S, x⁻¹ ∈ algebra.adjoin F S) :
```

Comment:
I guess in a future PR we could also have a version that asks for all `x \in S` to be algebraic...

Reply:
I have most of this (in the form `subalgebra.is_field_of_is_algebraic`) in the Dedekind domain branch, and will get around to PR'ing it soon.